packages/account/src/Sections/Verification/ProofOfOwnership/__test__/ExpandedCard.spec.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import ExpandedCard from '../ExpandedCard.jsx';
+
+const formatIdentifier = (id, type) => {
+    let formattedID = id;
+    if (type === 'IcCreditCard' || type === 'IcStockVisa' || type === 'IcStockMasterCard')
+        formattedID = `${id.substr(0, 6)}XXXXXX${id.substr(12)}`;
+    else if (type === 'IcEwallet') return formattedID;
+    return formattedID
+        .replace(/\s/g, '')
+        .replace(/(\w{4})/g, '$1 ')
+        .trim();
+};
```

Comment:
Seems like a duplicate for `packages/account/src/Sections/Verification/ProofOfOwnership/ExpandedCard.jsx`. Probably can extract it out. But since this is a test the best way is to have a `fixed expected value`